src/borg/archive.py

```diff
@@ -399,6 +399,7 @@ def info(self):
                 'hostname': self.metadata.hostname,
                 'username': self.metadata.username,
                 'comment': self.metadata.get('comment', ''),
+                'chunker_params': self.metadata.get('chunker_params', ''),
```

Comment:
looks good.
but i guess the docs about the json format also need an update about this?

Reply:
> but i guess the docs about the json format also need an update about this?
I hope I found the right place.. :-)